Src/Newtonsoft.Json/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,104 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
```

Comment:
@JamesNK Thanks for fixing the build by updating VS. I added some comments to explain he breaking changes that were found in the package as it is today. Could you please take a look and confirm all of them are by design?

Reply:
They look correct.